tests/app/views/test_application.py

```diff
@@ -41,6 +41,37 @@ def test_approval(sample_submitted_application):
         six.viewitems(a.data['recruiter_info'])
 
 
+def test_approval2(sample_submitted_application2):
```

Comment:
I thought test_approval would be run for each data file?  no need to create a new test?